src/stats.c

```diff
@@ -1109,6 +1120,10 @@ stats_general_print(emitter_t *emitter) {
 	OPT_WRITE_BOOL("prof_leak")
 	OPT_WRITE_BOOL("stats_print")
 	OPT_WRITE_CHAR_P("stats_print_opts")
+	OPT_WRITE_BOOL("stats_print")
+	OPT_WRITE_CHAR_P("stats_print_opts")
+	OPT_WRITE_UINT64("stats_interval")
```

Comment:
I think it now should be `OPT_WRITE_INT64`?

Reply:
Good catch! Let me do another round of search to make sure.